libs/saksbehandling-common/src/testFixtures/kotlin/etterlatte/behandling/Saksider.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.etterlatte.behandling
+
+import com.fasterxml.jackson.databind.JsonNode
+import kotlin.random.Random
+
+val sakId1 = tilSakId(1L)
+val sakId2 = tilSakId(2L)
+val sakId3 = tilSakId(3L)
+
+fun randomSakId() = tilSakId(Random.nextLong(10, Int.MAX_VALUE.toLong()))
+
+fun tilSakId(long: Long) = long
+
+fun tilSakId(int: Int) = tilSakId(int.toLong())
+
+fun JsonNode.tilSakId() = tilSakId(this.asLong())
```

Comment:
Antar dette bare er midlertidig og vil bli erstattet med direkte bruk av `SakId(long)` i neste steg 😄

Reply:
Satsar på det for tilSakId-metodane, iallfall.